draft-trammell-taps-interface.md

```diff
@@ -361,6 +361,13 @@ transport protocol selection is necessarily tied to path selection. This may
 involve choosing between multiple local interfaces that are connected to
 different access networks.
 
+In case of conflicts between protocol and path selection properties,
+path selection properties take precedence.
+For example, if an application indicates a preference for a specific path, but
+also a preference for a protocol not available on this path, the transport
+system will try the path first, so the protocol selection property might not
+have an effect.
+
```

Comment:
Precedence order (as above) should IMO appear here too...

Reply:
Agreed, I added some text here too, and moved it below the table. This section should perhaps be restructured in the future.